### DISCUSSION TITLE
Add an option to have the library as home page

### DISCUSSION THREAD

**Discussion by nalmada** [ChristopherKlay/StadiaEnhanced#130] [Ideas & Requests]
Would be nice to have an option to have the library page (https://stadia.google.com/library) as homepage

The workflow could be : 

If the option is enabled :

- Launch stadia (`stadia.google.com`), Stadia will redirect you to `https://stadia.google.com/home`. Add a redirection to `https://stadia.google.com/library`
- From any page, if you click on the `Home` button on top bar, go directly to `https://stadia.google.com/library`

TIA

**Comment by ChristopherKlay** [comment]
Initially i was going to show the library on both the homescreen and library page, effectively restoring how the homescreen worked pre-change. A forced redirect (which would require a page reload) wouldn't be a smooth solution sadly.

Looking at the changes made by Google now and the changes coming in the future however, this wouldn't make that much sense anymore, due to the library having it's own dedicated page and the homescreen likely more being used as a dashboard (a quick overview of many smaller things) in the future - not a single page where everything is dumped into anymore.
